content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -2,9 +2,19 @@
 title: Publishing Experimental Plugin Releases
 layout: developer
 ---
+The link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Manager tool] downloads plugins and their dependencies into a folder, which enables easier import into an instance of Jenkins.
+This benefits the user by providing more information about the plugins being downloaded, such as available updates or security warnings and many more.
 
-To simplify delivery of beta versions of plugins to interested users, the Jenkins project publishes an _experimental update center_.
-It includes alpha and beta versions of plugins, which aren't usually included in the regular update center.
+To simplify delivery, link:/doc/developer/plugin-development/incrementals/[Incremental] builds can be used. 
+This provides the user with flexibility to create incremental versions of Jenkins core or an upstream plugin on which under development downstream plugin can depend upon. 
```

Comment:
```suggestion
To simplify delivery, link:/doc/developer/plugin-development/incrementals/[incremental] builds can be used. 
This method provides the user with flexibility to create incremental versions of Jenkins core or an upstream plugin with dependencies. 
```
Suggestion for changing the link alt text to sentence case
The other suggestion is to rephrase this sentence as the last part is a bit awkward. I've suggested changing it to dependencies as I think that would mean the same thing in this case. However, if there are other alternatives, I'm very open to it!

Reply:
what does the sentence actually say? It sounds a bit like it's trying to explain what incremental means, using the word incremental. I think this page should not try to explain incrementals at all (just link to https://www.jenkins.io/doc/developer/plugin-development/incrementals/ where needed), instead it should clearly explain why are incrementals considered better than experimental update center releases both for testing and for downstream plugin development.